DDSim/DDSim/Helper/LCIO.py

```diff
@@ -0,0 +1,20 @@
+"""Helper object for files containing one or more MCParticle collections"""
+
+from DDSim.Helper.Input import Input
+
+class MCParticles( Input ):
```

Comment:
Sorry I wasn't clear, the class should also be called LCIO.
Then you get the "--LCIO.mcParticleCollectionName" command line flag and option named "LCIO...." in the steering file.

Conceivably we get more options for lcio files at some point.